components/AppForm/validates.js

```diff
@@ -0,0 +1,21 @@
+const required = values => {
```

Comment:
```suggestion
const required = value => {
```

Estas validaciones se hacen por campo, luego reciben un unico valor.